framework/src/modules/interoperability/schemas.ts

```diff
@@ -94,6 +91,28 @@ export const ccmSchema = {
 	},
 };
 
+const activeChainValidatorsSchema = {
+	type: 'array',
+	items: {
+		type: 'object',
+		required: ['blsKey', 'bftWeight'],
+		properties: {
+			blsKey: {
+				dataType: 'bytes',
+				fieldNumber: 1,
+				minLength: BLS_PUBLIC_KEY_LENGTH,
+				maxLength: BLS_PUBLIC_KEY_LENGTH,
+			},
+			bftWeight: {
+				dataType: 'uint64',
+				fieldNumber: 2,
+			},
+		},
+	},
+	minItems: 1,
+	// maxItems: MAX_NUM_VALIDATORS,
```

Comment:
I did that, because this schema is used on both mainchain and sidechain and they have different value in `maxItems`. So the actual `maxItems` is stated accordingly inside the `xxxRegParams` schema

Reply:
https://github.com/LiskHQ/lisk-sdk/blob/745154e81c8cbdef6320c525db3a1bc436e3bb0c/framework/src/modules/interoperability/mainchain/module.ts#L273

This validator function would verify number of items by reading `maxItems` keyword
https://ajv.js.org/json-schema.html#keywords-for-arrays

Reply:
Ok I  think I understand now. Is this because `genesisInteroperabilitySchema` in `chainInfos.chainValidators` uses the [`chainValidatorsSchema`](https://github.com/LiskHQ/lisk-sdk/blob/0f78514b5d1cae959dc44efa82b3c856bffab020/framework/src/modules/interoperability/stores/chain_validators.ts#L24) which has this `MAX_NUM_VALIDATORS` set?

Reply:
Yes you are correct